src/planning/plan/tests/planning-result-test.ts

```diff
@@ -39,19 +39,19 @@ describe('planning result', () => {
     const runtime = new Runtime({loader, context, memoryProvider});
     const arc = runtime.newArc('demo', storageKeyPrefixForTest());
     VolatileStorageDriverProvider.register(arc);
-    const storageService = arc.storageService;
-    const suggestions = await StrategyTestHelper.planForArc(arc);
-
-    assert.isNotEmpty(suggestions);
-    const result = new PlanningResult({context, loader, storageService});
-    result.merge({suggestions}, arc);
-
-    const serialization = result.toLiteral();
-    assert(serialization.suggestions);
-    const resultNew = new PlanningResult({context, loader, storageService});
-    assert.isEmpty(resultNew.suggestions);
-    await resultNew.fromLiteral({suggestions: serialization.suggestions});
-    assert.isTrue(resultNew.isEquivalent(suggestions));
+    // const storageService = arc.storageService;
```

Comment:
Add a TODO linked to bug to re-enable? Also, maybe we should just `describe.skip` at line 25 instead of the block comment (cleaner and more representative of what's happening).

Reply:
done